app/src/main/java/com/parking_system_kotlin/mvp/model/ParkingReservationModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.parking_system_kotlin.mvp.model
+
+import com.parking_system_kotlin.database.ReservationDatabase
+import com.parking_system_kotlin.entity.Reservation
+import com.parking_system_kotlin.mvp.contracts.ParkingReservationContract
+
+class ParkingReservationModel(private val database: ReservationDatabase = ReservationDatabase().getInstance()) :
```

Comment:
please do not put the `= ReservationDatabase().getInstance()` here. if you do this you may have bugs in the future

Reply:
done